problems/removeVowels.js

```diff
@@ -6,8 +6,16 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
+function removeVowels(str) {
+    let arr = str.split("")
+    let out = [];
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
This looks like it will work but we're doing more work than we need to. Instead we can iterate through our string (prefer an for of loop b/c we don't care about index) and our output can be "". Then we can use the includes method to see if "aeiouAEIOU".includes(char) . Try refactoring this :-)